packages/lesswrong/components/comments/NewUserGuidelinesDialog.tsx

```diff
@@ -0,0 +1,112 @@
+import Button from '@material-ui/core/Button';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import React from 'react';
+import { Link } from '../../lib/reactRouterWrapper';
+import { useNewEvents } from '../../lib/events/withNewEvents';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { useUpdateCurrentUser } from '../hooks/useUpdateCurrentUser';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    padding: theme.spacing.unit*2,
+    position:"relative"
+  },
+  assistance: { //UNUSED
+    color: theme.palette.text.normal,
+  },
+  'easy-going': {
+    color: theme.palette.text.moderationGuidelinesEasygoing,
+  },
+  'norm-enforcing': {
+    color: theme.palette.text.moderationGuidelinesNormEnforcing,
+  },
+  'reign-of-terror': {
+    color: theme.palette.text.moderationGuidelinesReignOfTerror,
+  },
+  'editButton': {
+    cursor: "pointer",
+    position: 'absolute',
+    right: 16,
+    height: '0.8em'
+  },
+  collapse: {
+    display:"flex",
+    justifyContent:"flex-end",
+    fontSize: 14,
+    marginBottom: 4,
+  },
+  moderationGuidelines: {
+    fontSize: "1.1rem",
+    '& p, & ul': {
+      marginTop: '.6em',
+      marginBottom: '.6em'
+    },
+    '& li': {
+      marginTop: '.4em',
+      marginBottom: '.4em'
+    }
+  }
+})
+
+const NewUserGuidelinesDialog = ({classes, onClose, post, user}: {
+  classes: ClassesType,
+  onClose: () => void,
+  post: PostsMinimumInfo,
+  user: UsersCurrent
+}) => {
+  const { LWDialog } = Components;
+  const updateCurrentUser = useUpdateCurrentUser();
+  const { recordEvent } = useNewEvents();
+
+  const handleClick = (e: React.MouseEvent) => {
+    e.preventDefault()
+    e.stopPropagation()
```

Comment:
what does this do? (for my own education)

Reply:
This was originally copied from `ModerationGuidelinesBox`, where I think it was serving some purpose, but it isn't necessary now, so removed.  Thanks for the catch